openedx/core/djangoapps/course_apps/models.py

```diff
@@ -1,3 +1,35 @@
 """
-Models for course apps.
+Models to store course apps data.
 """
+from typing import Dict
+
+from django.db import models
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import CourseKeyField
+from opaque_keys.edx.keys import CourseKey
+from simple_history.models import HistoricalRecords
+
+
+class CourseAppStatus(TimeStampedModel):
+    """
+    CourseAppStatus stores the enabled/disabled status for course apps.
+
+    .. no_pii:
+    """
+
+    app_id = models.CharField(max_length=32, db_index=True)
+    enabled = models.BooleanField(default=False)
+    course_key = CourseKeyField(max_length=255, db_index=True)
+
+    history = HistoricalRecords()
+
+    @classmethod
+    def get_app_status_data_for_course(cls, course_key: CourseKey) -> Dict[str, bool]:
```

Comment:
seems like this method is returning _all_ course apps? I found this name a little misleading by "get_app_status_for_course" rather I should probably be something `get_all_apps_status_for_course`

Reply:
also, can you add docstring please?